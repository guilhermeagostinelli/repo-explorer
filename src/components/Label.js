import styled from "styled-components";

const Label = styled.label`
  & > input,
  & > select {
    margin-left: 10px;
  }
`;

export default Label;
