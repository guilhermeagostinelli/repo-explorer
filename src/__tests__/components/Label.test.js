import React from "react";
import { create } from "react-test-renderer";
import Label from "../../components/Label";
import "jest-styled-components";

describe("<Label />", () => {
  it("should match the snapshot", () => {
    const label = create(<Label />);
    expect(label.toJSON()).toMatchSnapshot();
  });
});
