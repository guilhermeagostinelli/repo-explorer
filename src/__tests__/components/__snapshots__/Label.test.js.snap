// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> should match the snapshot 1`] = `
.c0 > input,
.c0 > select {
  margin-left: 10px;
}

<label
  className="c0"
/>
`;
